npm-crawl.js

```diff
@@ -552,20 +556,17 @@ utils.extend(FetchTask.prototype, {
 		// If a task is currently loading this fileUrl,
 		// wait for it to complete
 		var loadingTask = this.context.loadingPaths[this.fileUrl];
-		if(loadingTask) {
-			var task = this;
-			return loadingTask.promise.then(function(){
-				if(loadingTask.hadErrorLoading()) {
```

Comment:
Looks like this was removed but I don't think it should be. If there was a loading error that wasn't because of the wrong version, shouldn't we conclude that task2 will fail for the same reason and go ahead and error?